mobile/lib/models/article_info.dart

```diff
@@ -2,30 +2,37 @@
 class ArticleInfo {
   /// Creates an empty [ArticleInfo] object.
   ArticleInfo.empty()
-      : title = '',
+      : id = -1,
+        title = '',
         content = '',
         thumbnail = '',
         fullImage = '',
         imageTag = '';
 
   /// Creates a [ArticleInfo] object from a given Json map.
   ArticleInfo.fromJson(Map<String, dynamic> jsonData)
-      : title = (jsonData['title'] as Map<String, dynamic>)['rendered'] as String,
-        content =
-            (jsonData['content'] as Map<String, dynamic>)['rendered'] as String,
-        thumbnail = jsonData['_embedded']['wp:featuredmedia'][0]['media_details']['sizes']['thumbnail']['source_url'] as String,
-        fullImage = jsonData['_embedded']['wp:featuredmedia'][0]['media_details']['sizes']['full']['source_url'] as String,
+      : id = jsonData['id'] as int,
+        // ignore: avoid_dynamic_calls
+        title = jsonData['title']['rendered'] as String,
+        // ignore: avoid_dynamic_calls
+        content = jsonData['content']['rendered'] as String,
+        // ignore: avoid_dynamic_calls
+        thumbnail = jsonData['_embedded']['wp:featuredmedia'][0]
+            ['media_details']['sizes']['thumbnail']['source_url'] as String,
+        // ignore: avoid_dynamic_calls
+        fullImage = jsonData['_embedded']['wp:featuredmedia'][0]
+            ['media_details']['sizes']['full']['source_url'] as String,
         imageTag = jsonData['featured_media'].toString();
```

Comment:
I'm not sure why the `imageTag` is included here since it doesn't seem to be used. The image is already being loaded from `fullImage` and `thumbnail`.

Reply:
I think it was a remainder from merge with master, it can be removed